examples/web-auth.md

```
@@ -0,0 +1,725 @@
+```swift
+Auth0
+    .webAuth()
+    .provider(WebAuthentication.safariProvider()) // Use SFSafariViewController
+    .start { result in
+        // ...
+    }
+```
+
+> [!TIP]
+> See [`ASWebAuthenticationSession` vs `SFSafariViewController` (iOS)](https://auth0.github.io/Auth0.swift/documentation/auth0/useragents) to help determine which option best suits your use case, depending on your requirements.
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)